webapp/src/components/CollectionTable.vue

```diff
@@ -4,6 +4,7 @@
     :data="collections"
     :data-type="'collections'"
     :global-filter-fields="['collection_id', 'title', 'creatorsList']"
+    :edit-page-route-prefix="collections"
```

Comment:
Should be a string ?

```suggestion
    :edit-page-route-prefix="'collections'"
```